scripts/validate_docstrings.py

```diff
@@ -264,8 +277,7 @@ def examples_errors(self):
         error_msgs = ''
         for test in finder.find(self.raw_doc, self.method_name, globs=context):
             f = StringIO()
-            with contextlib.redirect_stdout(f):
-                runner.run(test)
```

Comment:
This change is for py27 compat ? 
As I seem to remember that this caused another error (@toobaz do you recall?)

Reply:
Yes it is for compat. I think the other error you are referring to was present in the very first commit 4673475ec7098a7b4203ca716cdda8c029618fc6 but this is a little different and should pass for 2 and 3